docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,196 @@
+---
+id: adjusting-bpmn-models
+title: Adjusting BPMN models
+description: "Learn how to adjust your BPMN models when migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+To migrate BPMN process models from Camunda Platform 7 to Camunda Platform 8, you must adjust them:
+
+- The namespace of extensions has changed (from `http://camunda.org/schema/1.0/bpmn` to `http://camunda.org/schema/zeebe/1.0`)
+- Different configuration attributes are used
+- Camunda Platform 8 has a _different coverage_ of BPMN elements (see [Camunda Platform 8 BPMN coverage](/components/modeler/bpmn/bpmn-coverage.md) vs [Camunda Platform 7 BPMN coverage](https://docs.camunda.org/manual/latest/reference/bpmn20/)), which might require some model changes. Note that the coverage of Camunda Platform 8 will increase over time.
+
+The following sections describe what the existing community supported [Backend diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter) does by BPMN symbol and explain unsupported attributes.
```

Comment:
Do the docs really rely on the coverage of the community tool or should we explain how concept / element x can be realized in the C8 context?

Reply:
Great question - I would love to avoid relying on community tools since it can be perceived as risky. 

Can you provide more info or content on this topic? This is a great backlog topic.

Reply:
Follow up issue here - https://github.com/camunda/camunda-platform-docs/issues/2371